docker/dkeycache/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
```

Comment:
combine and named as "start_dkeycache.sh"

Reply:
It will be fixed